content/docs/install.md

```diff
@@ -1,86 +1,94 @@
-    * Note: If an atomic.h error occurs, also add `libx32stdc++-12-dev`
-    * For FreeBSD: `pkg install bash git llvm14`
-    * Make sure the LLVM binaries and the linker are added to your `$PATH` environmental variable
+**Note:** If you get the error: `'linux/futex.h' file not found`, try installing `linux-headers`.
 
-## Building Odin
-Now, it's time to build Odin and get started!
+**Note:** On some distributions, you need to add a "-dev" suffix to the llvm and clang packages like so: `llvm14-dev`, `clang14-dev` to get the development files.
 
-### For Windows
-There's a couple prerequisites here. First, make sure you have Visual Studio installed; you have to compile Odin from source, and Odin also requires `link.exe` from VS anyway. The necessary LLVM components for Windows are included in the Odin repository.
+**Note:** The package names may differ slightly depending on your distribution.
 
-Now, it's time to build Odin and get started! Open the X64 Visual Studio command prompt ([if you don't typically use it, here's how to find it](https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt-for-vs)) and navigate to the directory where you downloaded Odin. Run the `build.bat` file, and you should have a successfully built Odin compiler!
+### BSD
+- On FreeBSD, run: `pkg install bash git llvm14`
+- [Add the LLVM binaries to your `$PATH`.](#add-llvm-to-your-path)
+- Navigate to the Odin directory inside your terminal.
+- Run `make` to build Odin. That creates the `odin` binary.
+- [Add the Odin directory to your `$PATH`.](#add-odin-to-your-path)
+- Use `odin --help` to learn how to run your first Odin program.
 
-To use Odin `link.exe` is required to be in the PATH of the callee as mentioned, this can either be achieved but calling Odin from the X64 Visual Studio command prompt or by calling the vcvarsall.bat (with x64 as an argument) script either in your shell or in your build script.
+## Add LLVM to your PATH
+To add LLVM to the `$PATH` environment variable, insert the line: `export PATH=$PATH:/path/to/llvm/bin` in your shell configuration file. This file may be called `~/.bashrc` or `~/.zshrc` or `/etc/profile`; it depends on your shell and platform. Then apply the change to the current session by running: `source ~/.name_of_shell_config_file`. Alternatively, you can restart the terminal to apply the change.
 
-### For MacOS
-Make sure all requirements for MacOS are installed, after installing LLVM through Homebrew make sure to add it to the PATH:
-- run `echo 'export PATH="/usr/local/opt/llvm/bin:$PATH"' >> ~/.zshrc_profile` to add LLVM to your PATH.
+**Note:** The installed location of LLVM depends on your platform. Two possible locations to look into are: `/usr/local/opt/llvm/bin` and `/usr/lib/llvm14/bin`.
 
-Then run `source ~/.bash_profile` or `source ~/.zshrc` to update your PATH variable in the current terminal session depending on your shell.
+## Add Odin to your PATH
```

Comment:
This is not necessary, but can be useful. Make that clear.

Reply:
I made this information a small tip instead.